spark/src/main/java/org/apache/iceberg/spark/data/SparkOrcWriter.java

```diff
@@ -42,7 +42,7 @@
  * This class acts as an adaptor from an OrcFileAppender to a
  * FileAppender&lt;InternalRow&gt;.
  */
-public class SparkOrcWriter implements OrcValueWriter<InternalRow> {
+public class SparkOrcWriter implements OrcRowWriter<InternalRow> {
 
   private final Converter[] converters;
```

Comment:
Does it make sense to file a ticket to refactor SparkOrcWriter to utilize similar approach of using `OrchSchemaVisitor`. This could help with reuse of common generic writers defined in `GenericOrcWriters`, similar to how we did in `SparkOrcReaders`?

Reply:
Thankyou!